src/utils/general.jl

```diff
@@ -198,20 +198,34 @@ end
 (processor::LSPostProcessor)(md::Markdown.MD) = processor.inner(lsrender(md))
 (processor::LSPostProcessor)(s::AbstractString) = processor.inner(s)
 
-function app_notfound_message(app::AbstractString, instruction_url::AbstractString)
+function install_instruction_message(app::AbstractString, instruction_url::AbstractString)
     """
-    `$app` executable is not found on the `PATH`.
     Follow this [instruction]($instruction_url)
     to install the `$app` app.
     """
 end
-function app_notfound_message(app::AbstractString)
+
+function check_settings_message(setting_path::String...)
+    """
+    This value is configured in the `$(join(setting_path, "."))`.
+    Please check the settings.
+    """
+end
+
+function app_notfound_message(app::AbstractString, is_default_setting::Bool=true, setting_path::String...)
```

Comment:
Wow, I didn't know this can be a valid method signature.
```suggestion
function app_notfound_message(app::AbstractString, setting_path::String...; is_default_setting::Bool=true)
```
seems to be more straightforward?